async_firebase/messages.py

```diff
@@ -4,12 +4,25 @@
 
 import typing as t
 from dataclasses import dataclass, field
+from enum import IntEnum
 
 import httpx
 
 from async_firebase.errors import AsyncFirebaseError
 
 
+class Visibility(IntEnum):
+    """Available visibility levels.
+
+    To get more insights please follow the reference
+    https://developer.android.com/reference/android/app/Notification#visibility
+    """
+
+    PRIVATE: int = 0
+    PUBLIC: int = 1
+    SECRET: int = -1
```

Comment:
Curious, why a negative integer here?

Reply:
Those are the numbers defined by Google. Here is the [link ](https://developer.android.com/reference/android/app/Notification#VISIBILITY_PRIVATE) to documentation